falcon/response.py

```diff
@@ -771,6 +776,19 @@ def add_link(self, target, rel, title=None, title_star=None,
         if anchor is not None:
             value += '; anchor="' + uri_encode(anchor) + '"'
 
+        if crossorigin is not None:
+            crossorigin = crossorigin.lower()
+            if crossorigin not in _RESERVED_CROSSORIGIN_VALUES:
+                raise ValueError(
+                    'crossorigin must be set to either '
+                    "'anonymous' or 'use-credentials'")
+            if crossorigin == 'anonymous':
+                value += '; crossorigin'
+            else:  # crossorigin == 'use-credentials'
+                # PERF(vytas): the only remaining value is inlined.
+                # Un-inline in case more values are supported in the future.
+                value += '; crossorigin="use-credentials"'
```

Comment:
IIRC, the double-quotes are unnecessary, but they don't hurt either.

Reply:
I was actually reading upon this one too :slightly_smiling_face: 

But in the the majority of examples in the wild used quotes, and @nagaabhinaya proposed this way as well, so I decided those two bytes were not worth saving.